infra/Pos/Slotting/Impl/Sum.hs

```diff
@@ -40,32 +40,48 @@ data SlottingContextSum
 
 -- | Monad which combines all 'MonadSlots' implementations (and
 -- uses only one of them).
-type MonadSlottingSum ctx m = (MonadReader ctx m, HasLens SlottingContextSum ctx SlottingContextSum)
+type MonadSlottingSum ctx m =
+    ( MonadReader ctx m
+    , HasLens SlottingContextSum ctx SlottingContextSum
+    )
 
 askSlottingContextSum :: MonadSlottingSum ctx m => m SlottingContextSum
 askSlottingContextSum = view (lensOf @SlottingContextSum)
 
-type SlotsSumEnv ctx m = (MonadSlottingSum ctx m, NtpMode m, SimpleSlottingMode m)
+type SlotsSumEnv ctx m =
+    ( MonadSlottingSum ctx m
+    , NtpMode m
+    -- ^ Contains @MonadThrow@
+    , SimpleSlottingMode m
+    )
 
-getCurrentSlotSum :: SlotsSumEnv ctx m => m (Maybe SlotId)
+getCurrentSlotSum
```

Comment:
Nah, I manually indented that since I find it more readable. I can revert it, but find it's more consistent and _easier to read_ if I have same indentation rule everywhere - not some "if it's 50 spaces wide than this, if it's 60 then...".